piwebapi_samples/Data Analysis/Sandbox.ipynb

```diff
@@ -0,0 +1,694 @@
+{
```

Comment:
Same comment as other *.ipynb files, do we actually need these checked in?

Reply:
This is the actual Jupyter Notebook. So this needs to be checked in. I think we can remove the other checkpoint.ipynb file as that's created automatically when you checkpoint your original notebook.